fineract-provider/src/main/resources/db/changelog/tenant/parts/0027_add_loan_cob_job_data.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Something has been mixed up I think and this file is not needed. Please double-check.